tasks/update_record.json

```diff
@@ -27,7 +27,7 @@
             "sensitive": true
         },
         "instance": {
-            "description": "ServiceNow instance name. If your instance uri is https://dev84270.service-now.com your instance name would be dev84270.",
+            "description": "ServiceNow instance name. If your instance uri is https://dev84270.service-now.com your instance name would be dev84270.service-now.com.",
```

Comment:
Hmm, should this be updated since we're no longer using instance names?

Reply:
updated.